Cartesian_kernel/include/CGAL/Cartesian/Point_3.h

```diff
@@ -45,6 +45,11 @@ class PointC3
   PointC3(const FT &x, const FT &y, const FT &z)
     : base(x, y, z) {}
 
+  PointC3(FT &&x, FT &&y, FT &&z)
+    : base(std::forward<FT>(x), std::forward<FT>(y), std::forward<FT>(z))
```

Comment:
It is a bit confusing to use std::forward here, std::move would be more natural. Or you could make the types of the 3 arguments template parameters and then using std::forward would be good.